src/components/Navbar/Navbar.tsx

```diff
@@ -14,9 +14,9 @@ export interface Props {
 const useStyles = makeStyles(() => ({
   icon: {
     fill: '#708284',
```

Comment:
should this fill be using a token color? (unrelated to your PR but still)

Reply:
Good call out.. no we need to get new token value from @chudo76 perhaps.
We can follow up separately maybe.

Reply:
Token for " x " should be `tw-clr-icn-secondary`. For Kaloko, that would map to `#7F775B`.

Reply:
Thanks @chudo76 !